docs/guide/input-validation.md

```diff
@@ -372,6 +373,14 @@ class MyForm extends Model
 }
 ```
 
+> Note: you can use [[yii\validators\InlineValidator::addError()]] for adding errors instead. That way the error
+> message can be formatted using [[yii\i18n\I18N::format()]] right away. Use `{attribute}` and `{value}` in the error
```

Comment:
@dynasource I left it as is, without leading backslash, to be the same as in the rest of the docs, also like @rob006 said it's not required.